build.gradle.kts

```
@@ -33,6 +33,8 @@ kotlin {
     linuxArm64()
     linuxArm32Hfp()
     linuxX64()
+    linuxMips32()
+    linuxMipsel32()
```

```suggestion
    linuxMips32()
    linuxMipsel32()
    linuxX64()
```
(The current list is alphabetized)

---

Well, kinda. I see I put 32 after 64 above 🙈

---

Done (if I got it correctly)